src/modules/shopPage/reducer.tsx

```diff
@@ -27,6 +27,7 @@ export type ShopItemAPI = {
   brand_id: number;
   size_id: number;
   supplier_id: number;
+  param: string;
```

Comment:
What that “param” for ?

Reply:
param - its data where we can see tires have or haven't studded and I need add it if I want see use this data
I use it in Characteristics component

Reply:
add note